lstchain/scripts/lstchain_longterm_dl1_check.py

```diff
@@ -792,8 +794,17 @@ class pixwise_info(tables.IsDescription):
 
     # We write out the camera geometry information, assuming it is the same
     # for all files (hence we take it from the first one):
-    subarray_info = SubarrayDescription.from_hdf(files[0])
-    subarray_info.to_hdf(output_file_name)
+    # subarray_info = SubarrayDescription.from_hdf(files[0])
+    # subarray_info.to_hdf(output_file_name)
+
+    cam_description_table = \
+        Table.read(files[0],
+                   path='/configuration/instrument/telescope/camera/geometry_0')
+    geom = CameraGeometry.from_table(cam_description_table)
```

Comment:
Yes, please just copy the whole instrument description. The code before was fine! Why change it?